src/librustc/middle/stability.rs

```diff
@@ -432,7 +432,7 @@ impl<'a, 'gcx, 'tcx> TyCtxt<'a, 'gcx, 'tcx> {
     // (See issue #38412)
     fn skip_stability_check_due_to_privacy(self, mut def_id: DefId) -> bool {
         // Check if `def_id` is a trait method.
-        match self.sess.cstore.describe_def(def_id) {
+        match ty::queries::describe_def::get(self, DUMMY_SP, def_id) {
```

Comment:
Ah, I guess that since @eddyb's #41504 has not landed yet, this is still the correct form. Perhaps though we want to name the query `def_of`, in anticipation of that PR's new conventions? (And this would then become `self.def_of(def_id)`.)